### DISCUSSION TITLE
Zephyrus G14 GA401QM 5900HS

### DISCUSSION THREAD

**Discussion by Heyth** [PIut02/ROG-Zephyrus-G14-GA401IV-Hackintosh#26] [General]
Hello, I came across your project and I have the Ryzen 5900HS model. I have been searching for a few days and this is the closest thing I have found. I was wondering if it was possible to create or modify the existing EFI.

**Comment by PIut02** [comment]
The 5900HS and 5800HS have the same number of cores, so you can use the 2021 folder directly, everything else is the same

**Reply by Heyth** [reply]
Hello, after much trial and error, I realized that I had my GPU mode set to standard instead of eco. When I switched to eco mode, the install worked fine. However, I still have issues with my audio and mic not being detected. I don’t know if this matters, but my model originally had a MediaTek WiFi card, which I replaced with an Intel one due to problems. ive also tried putting alcid=11 in boot args and that didnt work also 
[Codec.txt](https://github.com/PIut02/ROG-Zephyrus-G14-GA401-Hackintosh/files/13864818/Codec.txt)

**Reply by PIut02** [reply]
The network card is because ASUS has chosen to mix MTK and Intel models, but MTK does not work on hackintosh.
Codec shows that the model of the sound card is alc289, try to modify the value of `DeviceProperties > PciRoot(0x0)/Pci(0x8,0x1)/Pci(0x0,0x6) > layout-id` to 33 instead of adding boot-args.

**Reply by Heyth** [reply]
Thank you, it worked. But iMessage isn’t working because NullEthernet keeps defaulting the MAC address to 11:22:33:44:55:66. I don’t understand this guide or which method was used. [https://github.com/RehabMan/OS-X-Null-Ethernet](url)

**Reply by PIut02** [reply]
The problem is not in this place because we are using DSDT/SSDT method. Have you generated and saved your model serial number?

**Reply by Heyth** [reply]
Yes, I have. I’m still booting off the USB. I don’t know if that matters.
![Screenshot 2024-01-09 at 12 12 29 PM](https://github.com/PIut02/ROG-Zephyrus-G14-GA401-Hackintosh/assets/48037993/2b43d641-a144-47e1-9ad2-16f7168513c1)

**Reply by oceanondawave** [reply]
In my case I use `Ventura` because `Sonoma` has a lot of bugs. Wi-Fi and BlueTooth are not stable since `itlwm` has not released a stable version for it. Just reinstall your hackintosh using the `Ventura` configuration and replace your `AirportItlwm` kext with the `AirportItlwm` **v2.2.0 stable** for **Ventura** here: https://github.com/OpenIntelWireless/itlwm/releases. iMessage works for me.
